backend/src/database/seeds/04_contacts.js

```diff
@@ -0,0 +1,50 @@
+exports.seed = function (knex) {
```

Comment:
Just FYI here, if `seed()` needs to be a "blocking" operation - i.e. if you need to wait for seed to complete before doing something else in your code - then this won't work. Because this function returns a "thenable" it'll get processed on the async stack and operations after seed() will be executed first.